pkg/operator/ceph/cr_manager.go

```diff
@@ -161,7 +161,23 @@ func (o *Operator) addToManager(m manager.Manager, c *controllerconfig.Context,
 	return nil
 }
 
+func (o *Operator) validateCRDsName() error {
+	for _, crd := range o.resources {
+		var crdNameLength int = len(crd.Name)
+		if crdNameLength > 38 {
+			return errors.New("CRD name cannot be bigger than 38 characters")
```

Comment:
As discussed, having a single limit for all CRs may break some existing CRs if it's not actually a limit already hit with some K8s error. We may want to just address the original bug for the object store and only add a check for that CR length to start with.